.github/workflows/main.yml

```diff
@@ -30,3 +30,10 @@ jobs:
       run: |
         GOOS=${{ matrix.os }} GOARCH=${{ matrix.arch }} go build -o ./bin/kitctl ./cmd/kitctl/
       working-directory: substrate
+    - uses: docker/login-action@v1
```

Comment:
why do we need this for this PR ? Wrong commit got pushed to this PR. Please correct it.

Reply:
Testing this in my fork repo ,Unfortunately it seems was added to previous PR's ,Please neglect it.Thanks